projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/asm/b2c/asm.e2e-flaky.cy.ts

```diff
@@ -733,4 +778,19 @@ context('Assisted Service Module', () => {
       });
     });
   }
+
+  function getCurrentCartId(customerName, customerPwd) {
+    return new Promise((resolve, reject) => {
+      fetchingToken(customerName, customerPwd, false).then((res) => {
+        // get customerId of it
+        createCart(res.body.access_token).then((response) => {
+          if (response.status === 200) {
```

Comment:
I guess there is sth like response.OK instead of 200.

https://angular.io/api/common/http/HttpStatusCode